openedx/core/djangoapps/schedules/templates/schedules/edx_ace/recurringnudge_week1/email/body.html

```diff
@@ -0,0 +1,3 @@
+Dear {{student_name}},
```

Comment:
Why do we need a subfolder named "schedules" within a folder already named "schedules"?